src/components/AddItems.js

```diff
@@ -89,70 +92,85 @@ const AddItemsToList = () => {
             <p>
               {`You have ${normalizeString(
                 shoppingListItemName,
               )} in your shopping list already`}
             </p>
           ) : null}
-          <div>
-            <label>
-              Name
+          <div className="">
+            <div className="flex justify-center">
+              {/* <label className="text-black">Name of Item</label>  */}
+
               <input
                 type="text"
+                placeholder="Add Item..."
                 value={shoppingListItemName}
                 onChange={shoppingListItemNameHandler}
+                className="border  text-grey-darkest md:w-2/3 px-2 py-2 md:px-4 md:py-3 mb-8 rounded"
               />
-            </label>
+            </div>
           </div>
-          <fieldset>
-            How soon are you likely to buy it again?
-            <div>
+          <div className="flex justify-center md:mb-8 mb-4">
+            <p className="">How soon are you likely to buy it again?</p>
+          </div>
+          <div className="flex flex-col md:flex-row md:space-x-4 text-black md:mb-8 mb-4">
+            <div className="md:w-1/3 mb-4">
               <label>
-                Soon
                 <input
                   type="radio"
                   name="next_purchase"
                   value="7"
                   checked={daysLeftForNextPurchase === 7}
                   onChange={daysLeftForNextPurchaseHandler}
+                  className="mr-4"
                 />
+                Soon
               </label>
             </div>
-            <div>
+            <div className="md:w-1/3 mb-4">
               <label>
-                Kind of soon
                 <input
                   type="radio"
                   name="next_purchase"
                   value="14"
                   checked={daysLeftForNextPurchase === 14}
                   onChange={daysLeftForNextPurchaseHandler}
+                  className="mr-4"
                 />
+                Kind of soon
               </label>
             </div>
-            <div>
+            <div className="md:w-1/3 mb-4">
               <label>
-                Not soon
                 <input
                   type="radio"
                   name="next_purchase"
                   value="30"
                   checked={daysLeftForNextPurchase === 30}
                   onChange={daysLeftForNextPurchaseHandler}
+                  className="mr-4"
                 />
+                Not soon
               </label>
             </div>
-          </fieldset>
-          <button type="submit">Add Item</button>
+          </div>
+          <div className="flex justify-center mt-8">
+            <button
+              className="flex items-center border py-2 px-3 hover:shadow-hover justify-center rounded-md bg-white text-black shadow-bottom w-32 md:w-60"
+              type="submit"
+            >
+              Add Item
+            </button>
```

Comment:
I'd suggest that all buttons in the app have the same style. Here, the "Add Item" button gets a drop shadow when you over it, while the "Go to list" button gets highlighted when you hover over it. :slightly_smiling_face:

Reply:
Great, yes thanks for the suggestion! I've added the shadow on hover to all the buttons rather than the highlight